src/main/java/karuna/karuna_backend/visitor/message/domain/VisitorMessageService.java

```diff
@@ -10,9 +11,12 @@
 public class VisitorMessageService {
 
     private final VisitorMessageRepository visitorMessageRepository;
+    private final EmailSender emailSender;
 
     public VisitorMessageDto sendMessage(VisitorMessageCreateDto visitorMessageCreateDto) {
         VisitorMessage visitorMessage = VisitorMessageMapper.toModel(visitorMessageCreateDto);
-        return VisitorMessageMapper.toDto(visitorMessageRepository.save(visitorMessage));
+        VisitorMessage savedVisitorMessage = visitorMessageRepository.save(visitorMessage);
+        emailSender.sendEmail("HARDCODED-EMAIL@EMAIL.PL", "KARUN VISTOR MESSAGE", savedVisitorMessage.getBody(), null);
+        return VisitorMessageMapper.toDto(savedVisitorMessage);
```

Comment:
Correct me if I am wrong, but "HARDCODED-EMAIL" is the email that message should be sent to right? So as previously we need to centralize the point from which we want to well get our karuna target email address. VisitorMessageCreateDto should be updated so it requires subject as field as well.